TwitchClient/GamesListViewController.swift

```diff
@@ -1,108 +0,0 @@
-//
-//  GamesListViewController.swift
-//  TwitchClient
-//
-//  Created by Ahmed kotb on 4/7/17.
-//  Copyright © 2017 Ahmed Kotb. All rights reserved.
-//
-
-import UIKit
-import SVProgressHUD
-
-class GamesListViewController: UIViewController, WebserviceManagerDelegate, UITableViewDelegate, UITableViewDataSource {
-
-    
-    @IBOutlet weak var gamesTableView: UITableView!
-    
-    
-    var games = [Game]()
-   
-    var selectedGame = Game()
-    
-    override func viewDidLoad() {
-        super.viewDidLoad()
-
-        // Do any additional setup after loading the view.
-        
-        self.gamesTableView.isHidden = true
```

Comment:
Don't you wanna explain why you're doing this as a comment or by extracting it to a method?
Also don't use `self` unless you have to (usually in a closure or initializer).

Topics: clean code.